### ISSUE TITLE
Is there a way to shuffle music?

### ISSUE THREAD

**Issue by miketunes** [OmerTu/GoogleHomeKodi#342]
I've got voice commands working with Artist and Album, is there a way to shuffle those playlist when they are played?

**Comment by miketunes** [comment]
Just checking on this again.

**Comment by miketunes** [comment]
Any update on this?  Is it just the syntax since there's already a shuffle tv show command?

**Comment by keydon** [comment]
hmm the playback of artists is already shuffeled, isnt it?
but yes, I can add album shuffling aswell

**Comment by miketunes** [comment]
I don't think it is, though I think I'm giving up on using Kodi for music and going to try Apple Music instead.
